src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -30,8 +30,7 @@ internal static class LoopFilter
         //    10101010
         //
         // A loopfilter should be applied to every other 8x8 horizontally.
-        private static readonly ulong[] Left64X64TxformMask = new ulong[]
-        {
+        private static readonly ulong[] _left64X64TxformMask = {
             0xffffffffffffffffUL,  // TX_4X4
```

Comment:
```suggestion
            0xffffffffffffffffUL, // TX_4X4
```

Same for other comments